mmcv/ops/csrc/onnxruntime/onnxruntime_register.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
```

Comment:
It would be better to have consistent c++  `guard defines` in headers', just like existing code in the repo. Please refer to [here](https://google.github.io/styleguide/cppguide.html#The__define_Guard).